src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -151,20 +154,20 @@
     <Compile Include="System\Data\SqlTypes\SqlXml.cs" />
     <Compile Include="$(CommonPath)\System\NotImplemented.cs" />
     <Compile Include="$(CommonPath)\System\Data\Locale\Locale.cs">
-        <Link>System\Data\Locale\Locale.cs</Link>
+      <Link>System\Data\Locale\Locale.cs</Link>
```

Comment:
Now that `BasicFieldNameLookup` has been created for use in Common, is it necessary for FieldNameLookup.cs and the Locale-related files to live under $(CommonPath) anymore? In other words, could these files be moved back to being under SqlClient instead of Common, since they are only used by SqlClient?